calour/ms1_experiment.py

```diff
@@ -68,10 +68,6 @@ class MS1Experiment(Experiment):
     --------
     Experiment
     '''
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.heatmap_databases = ('gnps',)
-
```

Comment:
I think it is useful to setup the databases in the init because every time you create an ms1 experiment, we want the gnps interface to be active. same for ampliconexperiment - if you init one (from read or from pandas) we want the database to be active.
what is the problem with keeping the __init__() function?